examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7066087c-6947-4a50-a7fb-6ab71c004ccf",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Log a pytorch model\n",
+    "Note: you can also use this workflow for HuggingFace models, as they're torch models under the hood!\n",
```

Comment:
We can delete this note, actually it's kinda misleading to say HF model is equivalent to torch model. 

We might want to show how to log an HF model with code, could be a followup.